pkgs/development/ocaml-modules/ocaml-gettext/stub.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, buildDunePackage, ocaml_gettext, ounit }:
+
+buildDunePackage rec {
+
+  pname = "gettext-stub";
```

Comment:
What is the point of this package?

Reply:
It’s a library that is needed to build `libguestfs` and `virt-top`.

Reply:
How does it differ from the normal `gettext`? It looks identical to me.

Reply:
The `pname` is different. You can have a look to the source code of `buildDunePackage`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/ocaml/dune.nix#L16 and to its documentation: https://nixos.org/nixpkgs/manual/#sec-language-ocaml

Reply:
Oh, I didn't expect the `pname` to have an impact on the build result. Sorry for the noise. I'm not familiar with ocaml, only reviewing this to speed up the progress :)